index.d.ts

```diff
@@ -0,0 +1,63 @@
+declare module "moleculer-apollo-server" {
+	import { ServiceSchema } from "moleculer";
+	import { Config } from "apollo-server-core";
+	export {
+		GraphQLUpload,
+		GraphQLExtension,
+		gql,
+		ApolloError,
+		toApolloError,
+		SyntaxError,
+		ValidationError,
+		AuthenticationError,
+		ForbiddenError,
+		UserInputError,
+		defaultPlaygroundOptions,
+	} from "apollo-server-core";
+
+	export * from "graphql-tools";
+
+	export interface ApolloServerOptions {
+		path: string;
+		disableHealthCheck: boolean;
+		onHealthCheck: () => {};
+	}
+
+	export class ApolloServer {
+		createGraphQLServerOptions(req: any, res: any): Promise<any>;
+		createHandler(options: ApolloServerOptions): void;
+		supportsUploads(): boolean;
+		supportsSubscriptions(): boolean;
+	}
+
+	export interface GraphQLNodeResolver {
+		action: string;
+		rootParams?: {
+			[key: string]: string;
+		};
+		dataloader?: boolean;
+	}
+
+	export interface GraphQLTypeResolver {
```

Comment:
I have a similar concern here regarding the name seeming like its a native GraphQL type.  A good name eludes me right now, but maybe `ServiceResolverSchema` or `ServiceResolverDefinition` since it is what is passed to `createServiceResolvers`?

Reply:
Great catch. I love `ServiceResolverSchema`